src/components/widgets/CheckboxWidget.js

```diff
@@ -16,6 +15,14 @@ function CheckboxWidget(props) {
     onFocus,
     onChange,
   } = props;
+
+  // Because an unchecked checkbox will cause html5 validation to fail, only add
+  // the "required" attribute if the field value must be "true", due to the
+  // "const" or "enum" keywords
+  const required =
+    schema.const ||
```

Comment:
Do you think we need to put this, or is it unnecessary?:
```suggestion
    schema.const === true ||
```

Reply:
It feels a bit unnecessary, we only really want to check that `const` is truthy